docker/ngen.dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG ROCKYLINUX_TAG=8
+FROM rockylinux:${ROCKYLINUX_TAG}
+
+RUN dnf update -y \
+    && dnf install -y dnf-plugins-core epel-release \
+    && dnf repolist \
+    && dnf install -y --allowerasing tar git gcc-c++ gcc make cmake udunits2-devel coreutils \
+    && dnf clean all
+
+ARG BOOST_VERSION="1.72.0"
+RUN export BOOST_ARCHIVE="boost_$(echo ${BOOST_VERSION} | tr '\.' '_').tar.gz" \
+    && export BOOST_URL="https://boostorg.jfrog.io/artifactory/main/release/${BOOST_VERSION}/source/${BOOST_ARCHIVE}" \
```

Comment:
Per recent automation failures, this download URL is no longer usable. See the Sourceforge URL I switched stuff to in #699

Reply:
And note that the output-file arguments to curl may need to be adjusted as well.

Reply:
Ah, right. Thanks, will fix